Dockerfile

```diff
@@ -12,10 +12,21 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
-FROM golang:1.23-bullseye as builder
+FROM --platform=$BUILDPLATFORM golang:1.23-bullseye AS builder
+
+ARG TARGETOS
+ARG TARGETARCH
+
 WORKDIR /go/src/github.com/GoogleCloudPlatform/container-engine-accelerators
 COPY . .
-RUN go build cmd/nvidia_gpu/nvidia_gpu.go
+RUN if [ "${TARGETARCH}" = "arm64" ] && [ "${BUILDARCH}" != "arm64" ]; then \
+    apt update && \
+    apt install -yq --no-install-recommends \
+        gcc-aarch64-linux-gnu libc6-dev-arm64-cross; \
+        CC=aarch64-linux-gnu-gcc; \
+    fi && \
```

Comment:
This dockerfile is used to internally build image for release, also need to double check if the currently louhi pipeline can work successfully after this update.

Reply:
Yes, currently the Louhi pipeline runs `make container.` Locally `make container` performs the exact same as before, only using the dockerx wrapper around docker. For actually building the multi-arch build in louhi, I separated out that build into a different make command. I will create a seperate louhi flow to verify that works before proceeding with changing the pipeline.